### ISSUE TITLE
I am using React.js but I dont know why my props are not defined

### ISSUE THREAD

**Issue by Aakrity1105** [yannickcr/eslint-plugin-react#2695]
```jsx
import React, { Component } from 'react';
import logo from './logo.svg';
import './App.css';
import HttpService from '../services/http-service'

const http = new HttpService();

function App () {
	
	  constructor(props)
	{
		super(props);
		http.getProducts();
	}
	loadData = () => {
		http.getProducts().then(data => {
		},err => {
			
		})
	}
	
  return (
    <div className="App">
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <p>
          Edit <code>src/App.js</code> and save to reload.
        </p>
        <a
          className="App-link"
          href="https://reactjs.org"
          target="_blank"
          rel="noopener noreferrer"
        >
          Learn React
        </a>
      </header>
    </div>
  );
}

export default App;
```
![Screenshot from 2020-07-05 14-42-02](https://user-images.githubusercontent.com/67095481/86529353-b7d53d80-becd-11ea-9bfa-3ec35fcd2e6a.png)

**Comment by ljharb** [comment]
Your `function App` here is using invalid syntax; `constructor` is for `class`es, not for functions.

**Comment by Aakrity1105** [comment]
@ljharb I have posted that by mistake,now i have updated the correct code

**Comment by ljharb** [comment]
Setting aside the brace style, and that you shouldn't have functions in class fields - it still seems like you're not providing the full *unmodified* code that you're working with. Note that in a class component, you access props with `this.props`.

**Comment by Aakrity1105** [comment]
Actually I am new to github, still learning how to use it. I have by mistake created this issue. Can you delete this issue?

**Comment by ljharb** [comment]
I'm afraid I can't, but you're welcome to delete all your comments.
